.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,11 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+    tags:
+
+jobs:
+  ci:
+    uses: laminas/workflow-continuous-integration/.github/workflows/continuous-integration.yml@1.x
```

Comment:
I wonder if we can see a CI run on your fork?

Reply:
Typically, GH has picked these up in new PRs. I'm a bit suprised it's not showing it here. I'll see if it ran on my fork.

Reply:
Indeed, it did:

- https://github.com/weierophinney/getlaminas.org/actions/runs/2834213442

Reply:
Excellent! :ship: then :D